packages/grafana-ui/src/components/PanelChrome/PanelContext.ts

```diff
@@ -0,0 +1,8 @@
+import { EventBusWithSource } from '@grafana/data';
+import React from 'react';
+
+export interface PanelContextObject {
+  eventBus?: EventBusWithSource;
+}
+
+export const PanelContext = React.createContext<PanelContextObject>({});
```

Comment:
Not sure where to put the PanelContext, so it ended up here. Let me know if there's a better place for it.

Reply:
This makes sense @oscarkilhed